internal/command/ssh/sftp.go

```diff
@@ -509,6 +516,10 @@ func (sc *sftpContext) get(args ...string) error {
 		} else {
 			sc.out("wrote %d bytes", bytes)
 		}
+		err = f.Sync()
+		if err != nil {
+			sc.out("failed to sync %s: %s", localFile, err)
```

Comment:
Can we give the user more information about what happened and what to do about it?

Reply:
The `err` part should have what happened. E.g. "no space left on device" if this downloads 2GB file and the local host doesn't have the space.

"What to do about" is difficult. This error is not coming from Fly's platform. So we don't have much concrete suggestions.

Reply:
All good. 

I'd say these error messages are very normal. We tell the user "wrote %d bytes" which is technically true even if it can seem weird if the sync to disk then goes wrong. I was just wondering if we could very easily do better than "normal," while you've got a PR going already. I wonder this about most of our error messages. ;)